src/services/EdFi.DataManagementService.Api/ApiSchema/ApiSchemaLoader.cs

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+using System.Text.Json.Nodes;
+
+namespace EdFi.DataManagementService.Api.ApiSchema;
+
+/// <summary>
+/// Loads and parses the ApiSchema.json from a file.
+/// </summary>
+public class ApiSchemaFileLoader : IApiSchemaProvider
+{
+    private readonly JsonNode _apiSchemaRootNode;
+
+    /// <summary>
+    /// The parsed ApiSchema.json file
+    /// </summary>
+    public JsonNode ApiSchemaRootNode => _apiSchemaRootNode;
+
+    public ApiSchemaFileLoader(ILogger<ApiSchemaFileLoader> _logger)
+    {
+        _logger.LogDebug("Entering ApiSchemaFileLoader");
+
+        // Hardcoded and synchronous way to read the API Schema file
+        string fileLocation = $"{AppContext.BaseDirectory}/ApiSchema/DataStandard-5.0.0-ApiSchema.json";
+        JsonNode? rootNodeFromFile = JsonNode.Parse(File.ReadAllText(fileLocation));
```

Comment:
This might be difficult to work with from a unit testing perspective.

Maybe helpful to have a lightweight utility intereface & class that replaces File.ReadAllText.

Or alternately, could change the `fileLocation` to a protected property. In the unit tests, override the value of `fileLocation` to something specific for the unit test project.

Reply:
I have this injected as an IApiSchemaProvider, which only has a JsonNode get property. So unit tests can just be given a JsonNode directly.

Reply:
Right, that was my thought